openrarity/utils/data.py

```diff
@@ -17,17 +17,17 @@ def merge(
     Parameters
     ----------
     left : list[dict]
-        Data to be joined with
+        Data to be joined with.
     right : list[dict]
-        Data coming into the join
+        Data coming into the join.
```

Comment:
Worth specifying the this expects the `left` to be a longer list than `right` otherwise you will get data loss.